draft-packages/stories/Skirt.stories.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from "react"
```

Comment:
These stories are great - it gave me a good idea of anticipated usage.

Reply:
Now that these packages are consolidated, can you combine `Skirt.stories.tsx` and `PageLayout.stories.tsx` ?

Reply:
Yeah, actually they’re not _all_ consolidated yet (the skirt is a separate package). Can move those bits into `PageLayout` too and consolidate the stories if you think that’s the right move.

Reply:
My thoughts are that someone who is using this Skirt component would probably not use it in isolation, and instead would use it in conjunction with the page layout components. They seem related to me.

Reply:
Yep, fair. Have done that now.